deploy/kubernetes/charts/catalog/values.yaml

```diff
@@ -103,6 +103,15 @@ mysql:
 
   affinity: {}
 
+  persistentVolume:
+    enabled: false
+    annotations: {}
+    labels: {}
+    accessModes:
+    - ReadWriteOnce
+    size: 10Gi
+    storageClass: gp2
```

Comment:
I believe that the convention is to set this to an empty string so that it uses the default storage class of the cluster.